### DISCUSSION TITLE
OsmAnd-java tests error

### DISCUSSION THREAD

**Discussion by MarcosAndre28** [osmandapp/OsmAnd#14606] [Q&A]
when i try to build the project this error happens
![error teste](https://user-images.githubusercontent.com/34040590/174859975-f88cf106-f47e-44cc-bede-8f2e2eb0c738.png)

**Comment by vshcherb** [comment]
You can build without tests gradle build  -x test. Tests require thorough environment setup

**Reply by MarcosAndre28** [reply]
when I try to build without the tests, it shows a memory error in java
